typerighter.cfn.yaml

```diff
@@ -426,3 +446,26 @@ Resources:
           Bucket: !GetAtt CloudfrontBucket.DomainName
           IncludeCookies: 'false'
           Prefix: ''
+
+  RuleProvisionerAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: CreateProdAlarms
```

Comment:
This condition prevents the alarm from being created on CODE, remove this if you wish to test.